docs/Homebrew-brew-Maintainer-Guide.md

```diff
@@ -0,0 +1,102 @@
+# Homebrew/brew Maintainer Guide
+
+This document describes a few components of the `Homebrew/brew` repository that are useful for maintainers to
+be aware of, but don't necessarily need to appear in documentation for most users and contributors.
+
+## Reviewing PRs
+
+Using `gh pr checkout NUMBER` is a super easy way to check out a PR branch using the GitHub CLI.
+
+When reviewing a PR, use "comment", "approve", or "request changes" when submitting based on the following guidelines:
+
+- Comment: if the PR isn't quite ready to be merged
+- Approve: if you feel that the PR is in a good state to be merged, even if there are
+  non-blocking changes you'd like to be made
+- Request changes: if you feel strongly that the PR is likely to cause a problem for users or
+  have a genuine reason to oppose the PR.
+
+## Merging PRs
+
+Merging should be done using the standard Merge button in the `Homebrew/brew` repository to preserve history and GPG commit signing. The Squash and Merge and Rebase and Merge buttons are disabled.
+
+PRs must meet the following conditions to be merged:
+
+- Have at least one maintainer (or `BrewTestBot`) approval. See [below](#automatic-approvals)
```

Comment:
```suggestion
- Have at least one maintainer (or @BrewTestBot) approval. See [below](#automatic-approvals)
```

(and elsewhere) as iirc this should produce a clickable link.

Reply:
I can't remember if @-mentions in markdown are clickable but I'll test it and just make it a manual link if not

Reply:
In case you were curious: they don't work in markdown. And I'm pretty sure that it wouldn't transfer well to the docs page either. I'll do a manual link